presto-docs/README.md

```diff
@@ -0,0 +1,103 @@
+# presto-docs - Presto Documentation
+
+The presto-docs module contains the reference documentation for Presto.
+
+## Tools
+
+The default build of the docs is performed with Apache Maven.
+
+Documentation source files can be found in [Restructured
+Text](https://en.wikipedia.org/wiki/ReStructuredText) (`.rst`) format in
+`src/main/sphinx` and sub-folders.
+
+The engine used to create the documentation in HTML format is the Python-based
+[Sphinx](https://www.sphinx-doc.org).
+
+## Default Build
+
+The default build is using Apache Maven and Java like for the rest of the
+Presto build. You just need to have built the current version from the root.
+Subsequently you can build the site:
+
+```bash
+mvn -pl presto-docs clean install
+```
+
+or
+
+```bash
+cd presto-docs
+mvn clean install
+```
+
+This also performs other checks and it is the authoritative way to build the
+docs, however it is also considerably slower than using Sphinx directly.
```

Comment:
also it often hides errors -- @mosabua do you think it's worth mentioning it here?

Reply:
yes .. I will add that.

Reply:
done